configure.ac

```diff
@@ -42,6 +42,16 @@ debug_c_warn_flags="-Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing
 debug_c_other_flags="-fstack-protector-strong"
 picky_c_warn_flags="-Wno-long-long -Wmissing-prototypes -Wstrict-prototypes -Wcomment -pedantic"
 
+AC_ARG_WITH([build_id],
+	    [AC_HELP_STRING([--with-build_id],
+			    [Enable build_id annotations @<:@default=no@:>@])],
+	    [], [with_build_id=no])
+AS_IF([test x"$with_build_id" != x"no"],
+      [build_id=-"$with_build_id"],
+      [build_id=""])
```

Comment:
Might be simpler to:

```
AS_IF([test x"$with_build_id" = "no"], [with_build_id=""])
```

You really only want to set it back to empty if it was "no".

Reply:
Thanks